pkg/print/common/errors.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package common
+
+import "fmt"
+
+// ResultError is returned from printers when the apply/destroy operations completed, but one or
+// more resources either failed apply/prune/delete, or failed to reconcile.
+type ResultError struct {
```

Comment:
I don't love the name `common.ResultError` (too generic) but I don't really have a better suggestion.

Reply:
Agree. I have tried to come up with something better, but haven't found anything yet. Maybe something like `ActuationError` would be better?

Reply:
That would make the `failedActuateSum` variable name confusing. Does actuation include waiting for reconciliation or not?

Reply:
I was thinking yes in this case, but I agree that it could be confusing.